lib/gui/pages/main/controllers/image-selection.js

```diff
@@ -26,18 +26,26 @@ module.exports = function(SupportedFormatsModel, SelectionStateModel, AnalyticsS
    * @type {String[]}
    * @public
    */
-  this.mainSupportedExtensions = _.slice(SupportedFormatsModel.getAllExtensions(), 0, 3);
+  this.mainSupportedExtensions = _.intersection([
+    'img',
+    'iso',
+    'zip'
+  ], SupportedFormatsModel.getAllExtensions());
 
   /**
    * @summary Extra supported extensions
    * @constant
    * @type {String[]}
    * @public
    */
-  this.extraSupportedExtensions = _.difference(
+  this.extraSupportedExtensions = _.sortBy(_.difference(
     SupportedFormatsModel.getAllExtensions(),
     this.mainSupportedExtensions
-  );
+
+  // Sort alphabetically
+  ), (extension) => {
+    return extension.charCodeAt();
```

Comment:
Huh, why are you only sorting on the first character in the string? It means `abb` might sort before `aaa`.
Why not just use 'normal' sort? http://www.w3schools.com/jsref/jsref_sort.asp